ruby.html.markdown

```diff
@@ -276,6 +277,7 @@ end
 
 # You can also surround blocks in curly brackets:
 (1..5).each { |counter| puts "iteration #{counter}" }
+# although this is only done for one-liners.
```

Comment:
This _can_ be done even with multiline blocks, althogh it isn't encouraged. I think we could remove this comment.